app/AppCache.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
```

Comment:
__[code_sniffer]__
* Missing class doc comment
* Each class must be in a namespace of at least one level (a top-level vendor name)